Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM heroku/cedar:14
+
+ENV DEBIAN_FRONTEND noninteractive
+ENV LANG en_US.UTF-8
+ENV PATH $PATH:/root/.local/bin
+
+RUN mkdir -p /app/user
+WORKDIR /app/user
+
+RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 575159689BEFB442 \
+  && echo 'deb http://download.fpcomplete.com/ubuntu trusty main' > \
+    /etc/apt/sources.list.d/fpco.list \
+  && apt-get update \
+  && apt-get install -y \
+    coffeescript \
+    postgresql-client \
+    stack \
+  && apt-get clean \
+  && rm -rf /var/lib/apt/lists/*
+
+COPY stack.yaml .
+RUN stack setup
```

Comment:
I think it would be fine to install `postgres-client` in a generic Heroku image (or a thoughtbot image), since it's part of the core Heroku stack. It's also a fairly lightweight dependency.

I plan on extracting a more reusable Docker image from this once we get it working, but it's easier to develop everything with a simpler base image while it's still in flux.